app/views/admin/registers/_form.html.haml

```diff
@@ -0,0 +1,28 @@
+= form_for [:admin, @register], html: { autocomplete: "off" } do |f|
+  = f.text_field :name
+  = f.text_field :description
+  .form-group
+    = f.label :register_phase, class: 'form-label'
+    = f.select :register_phase, Register::CURRENT_PHASES, {}, include_blank: 'Select current phase', class: 'form-control'
+  .form-group
+    = f.label :authority, class: 'form-label'
+    = f.select :authority, @government_organisations.all.collect{ |org| [org.data['name'], org.key] }, {}, class: 'form-control'
+  = f.text_area :contextual_data
+  = f.text_area :related_registers
```

Comment:
It's unclear to me what you are expecting in this textarea. URLs? Markdown?

Reply:
They aren't used right now anyway so we can iterate on this at a later date. This entire admin interface will need some UCD input anyway but its low on priorities right now.

Reply:
maybe just remove these fields from the UI if we're not using them currently?